DigitalLearningSolutions.Data.Tests/Services/DiagnosticAssessmentServiceTests.cs

```diff
@@ -238,22 +235,137 @@ int progressId
             int progressId
         )
         {
-            using (new TransactionScope())
-            {
-                // Given
-                var scoresReturnedFromOldStoredProcedure = diagnosticAssessmentTestHelper
-                    .ScoresFromOldStoredProcedure(progressId, sectionId)
-                    .FirstOrDefault();
-
-                // When
-                var sectionInCourseContent = diagnosticAssessmentService
-                    .GetDiagnosticAssessment(customisationId, candidateId, sectionId);
-
-                // Then
-                sectionInCourseContent.SectionScore.Should().Be(scoresReturnedFromOldStoredProcedure.SecScore);
-                sectionInCourseContent.MaxSectionScore.Should().Be(scoresReturnedFromOldStoredProcedure.SecOutOf);
-                sectionInCourseContent.DiagnosticAttempts.Should().Be(scoresReturnedFromOldStoredProcedure.DiagAttempts);
-            }
+            // Given
+            var scoresReturnedFromOldStoredProcedure = diagnosticAssessmentTestHelper
+                .ScoresFromOldStoredProcedure(progressId, sectionId)
+                .FirstOrDefault();
```

Comment:
Why is `FirstOrDefault` necessary here? 🙂

Reply:
It didn't occur to be to change this after the comment for the post learning PR, I'll move this logic to the helper function.